docs/modules/ROOT/partials/supported-versions.adoc

```diff
@@ -2,4 +2,4 @@
 // This is a separate file, since it is used by both the direct Trino documentation, and the overarching
 // Stackable Platform documentation.
 
-- 362
```

Comment:
Ok it seems we drop support for Trino 362.
Just want to make sure if everybody is fine with that? I personally don't have a string opinion.

Reply:
Yes i discussed that with @lfrancke.